src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -11,33 +11,47 @@ import Label from '../Label';
 import { colors } from '../../index';
 
 const generateCityList = (amount: number): OptionProps[] => {
-  const finalData = [];
+  const tempData: string[] = [];
 
   for (let i = 0; i < amount; i += 1) {
-    const item = address.city();
-    finalData.push({
-      id: item.toLowerCase(),
-      optionValue: item,
-    });
+    let item = address.city();
+    while (tempData.includes(item)) {
+      item = address.city();
+    }
+    tempData.push(item);
   }
 
-  return finalData;
+  return tempData.map(item => ({
+    id: item.toLowerCase(),
```

Comment:
I think this would be slower, no?

```typescript
const generateCityList = (amount: number): OptionProps[] => {
  const finalData: OptionProps[] = [];

  for (let i = 0; i < amount; i += 1) {
    let city = address.city();
    let item = {
      id: city.toLowerCase(),
      optionValue: city,
    };
    while (finalData.includes(item)) {
      city = address.city();
      item = {
        id: city.toLowerCase(),
        optionValue: city,
      };
    }
    finalData.push(item);
  }

  return finalData;
};
```

Reply:
```js
const generateCityList = (amount: number): OptionProps[] => {
  const tempData: OptionProps[] = [];

  for (let i = 0; i < amount; i += 1) {
    let item = address.city();
    while (tempData.includes(item)) {
      item = address.city();
    }
    tempData.push(({
      id: item.toLowerCase(),
      optionValue: item,
    });
  }
};
```

Reply:
Just moving that final map into the tempDat push